core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -298,6 +317,39 @@ public static long defaultClientIdleTimeoutMillis() {
     }
 
     /**
+     * Returns the default maximum length of the initial line.
+     * Note that this value has effect only if a user did not specify it.
+     *
+     * <p>This default value of this flag is {@value #DEFAULT_DEFAULT_MAX_INITIAL_LINE_LENGTH}. Specify the
+     * {@code -Dcom.linecorp.armeria.maxInitialLineLength=<integer>} JVM option to override the default value.
+     */
+    public static int defaultMaxInitialLineLength() {
+        return DEFAULT_MAX_INITIAL_LINE_LENGTH;
+    }
+
+    /**
+     * Returns the default maximum length of all headers.
+     * Note that this value has effect only if a user did not specify it.
+     *
+     * <p>This default value of this flag is {@value #DEFAULT_DEFAULT_MAX_HEADER_SIZE}. Specify the
+     * {@code -Dcom.linecorp.armeria.maxHeaderSize=<integer>} JVM option to override the default value.
+     */
+    public static int defaultMaxHeaderSize() {
+        return DEFAULT_MAX_HEADER_SIZE;
+    }
+
+    /**
+     * Returns the default maximum length of the content or each chunk.
```

Comment:
... of each chunk in an HTTP/1 response content. The content or a chunk longer than this value will be split into smaller chunks so that their lengths never exceeds it.